addons/microsoft_calendar/models/microsoft_sync.py

```diff
@@ -364,6 +365,10 @@ def _sync_microsoft2odoo(self, microsoft_events: MicrosoftEvent):
         synced_recurrences |= cancelled_recurrences
         synced_events |= cancelled_events | cancelled_recurrences.calendar_event_ids
 
+        # Get sync lower bound days range for checking if old events must be updated in Odoo.
+        ICP = self.env['ir.config_parameter'].sudo()
+        lower_bound_day_range = int(ICP.get_param('microsoft_calendar.sync.range_lower_bound', default=365))
+
```

Comment:
people who just defined microsoft_calendar.sync.range_days expect it to continue working. We are in a stable branch.